raiden-dapp/src/components/FindRoutes.vue

```diff
@@ -102,6 +103,9 @@ export default class FindRoutes extends Vue {
   @Prop({ required: true })
   target!: string;
 
+  @Prop({ required: true })
+  pfs!: RaidenPFS | null;
```

Comment:
This propm doesn't seem to be used. If it's not null, it should be passed to `$raiden.findRoutes`, shouldn't it?

Reply:
Hm did we somehow break eslint or did it never scream about unused props/vars/...?